### ISSUE TITLE
Plugin abandoned / takeover?

### ISSUE THREAD

**Issue by WildRikku** [mdziekon/OctoPrint-SpoolManager#56]
The fork of the plugin listed on plugins.octoprint.org is marked as abandoned again. Are you still interested in maintaining this plugin? I noticed you put quite a lot of work into this repository, but I also noticed there have been no new commits to this repository in a long time.

**Comment by mdziekon** [comment]
See: https://github.com/mdziekon/OctoPrint-SpoolManager/discussions/53#discussioncomment-7987916

TL;DR - no, I'm not interested, I'm using Spoolman now on all my printers. Feel free to fork / cherry-pick my work, as long as my work has proper attribution.

**Comment by mdziekon** [comment]
@WildRikku 

See: https://github.com/mdziekon/OctoPrint-SpoolManager/discussions/53#discussioncomment-7987916

TL;DR - no, I'm not interested, I'm using Spoolman now on all my printers. Feel free to fork / cherry-pick my work, as long as my work has proper attribution.
